Jenkinsfile

```diff
@@ -148,7 +148,9 @@ pipeline {
             }
         }
         cleanup {
-            cleanWs()
+            script {
+                util.cleanNode()
```

Comment:
this will not clean `docker`, unless you specify a container engine as parameter
`util.cleanNode('docker')`
but we could update the shared lib method to clean `docker` by default.
I just put that as optional to avoid later problems with node without docker installed ...